packages/ncids-js/src/components/nci-feature-card-row/nci-feature-card-row.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * NCI Feature Card Row (3 Cards)
+ * Displays a row of 3 (on desktop) feature cards with image, title, description
+ * Displays cards stacked vertically with image inline and on the left for tablet
+ * Display cards stacked vertically with no image for mobile
+ */
+export class NCIFeatureCardRow {
+	/** The component that contains header section. */
+	protected element: HTMLElement;
+	/** The options for Header and MegaMenu. */
+	//protected options: NCIFeatureCardRowOptions;
+
+	/** Map object of the component. */
+	private static _components: Map<HTMLElement, NCIFeatureCardRow> = new Map<
+		HTMLElement,
+		NCIFeatureCardRow
+	>();
+
+	/**
+	 * Sets component variables and initializes component.
+	 *
+	 * @param {HTMLElement} element Component being created.
+	 * @param {NCIExtendedHeaderWithMegaMenuOptions} options Component being created.
+	 * @protected
+	 */
+	protected constructor(
+		element: HTMLElement
+		//options: NCIFeatureCardRowOptions
+	) {
+		this.element = element;
+		//this.options = options;
```

Comment:
is this section still a WIP?

Reply:
Do I even need this file if there isn't associated JS ?

Reply:
Nope, you can remove any associated ts files if there's no functionality